xgboost-2.0/recipe/0001-Use-central-cuda-capabilities.patch

```diff
@@ -1,23 +1,21 @@
-From 674b1e1eafd3ddbcabbd990d7da3c82eb539d851 Mon Sep 17 00:00:00 2001
-From: Deepali Chourasia <deepch23@in.ibm.com>
-Date: Tue, 8 Nov 2022 07:44:57 +0000
-Subject: [PATCH] use centralized cuda capabilities
+From 77cda4dbda989789a7bd8b3b568ac5130faca69e Mon Sep 17 00:00:00 2001
+From: Aman-Surkar <aman_surkar@persistent.com>
+Date: Wed, 4 Oct 2023 05:30:22 +0000
+Subject: [PATCH] Use central cuda capabilities
 
 ---
- cmake/Utils.cmake | 10 ++++++++--
- 1 file changed, 8 insertions(+), 2 deletions(-)
+ cmake/Utils.cmake | 8 ++++++++
+ 1 file changed, 8 insertions(+)
 
 diff --git a/cmake/Utils.cmake b/cmake/Utils.cmake
-index f28c1f27..f0b6f1b2 100644
+index 08050205..4ea974d7 100644
 --- a/cmake/Utils.cmake
 +++ b/cmake/Utils.cmake
-@@ -90,8 +90,13 @@ function(format_gencode_flags flags out)
-   endif()
-   # Set up architecture flags
+@@ -92,6 +92,13 @@ function(format_gencode_flags flags out)
    if(NOT flags)
--    if (CUDA_VERSION VERSION_GREATER_EQUAL "11.1")
--      set(flags "50;60;70;80")
-+    if (CUDA_VERSION VERSION_GREATER_EQUAL "11.4")
+     if (CUDA_VERSION VERSION_GREATER_EQUAL "11.8")
+       set(flags "50;60;70;80;90")
++    elseif (CUDA_VERSION VERSION_GREATER_EQUAL "11.4")
```

Comment:
Please add change to use $ENV{cuda_levels} in the cuda 11.8 block.

Reply:
You may update this in a follow up PR

Reply:
Update the patch.